aws/sdk/benchmarks/s3-express/benches/put_get_delete.rs

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use aws_config::BehaviorVersion;
+use aws_sdk_s3::primitives::ByteStream;
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use s3_express::{confidence_level, number_of_iterations, sample_size};
+use tokio::runtime::Runtime;
+
+pub fn put_get_delete(c: &mut Criterion) {
+    let buckets = if let Ok(buckets) = std::env::var("BUCKETS") {
+        buckets.split(",").map(String::from).collect::<Vec<_>>()
+    } else {
+        panic!("required environment variable `BUCKETS` should be set: e.g. `BUCKETS=\"bucket1,bucket2\"`")
+    };
+
+    let number_of_iterations = number_of_iterations();
+
+    println!(
+        "measuring {number_of_iterations} sequences of \
+            PutObject -> GetObject -> DeleteObject against \
+            {buckets:#?}, switching buckets on every sequence of operations\n"
+    );
+
+    let client = Runtime::new().unwrap().block_on(async {
+        let config = aws_config::load_defaults(BehaviorVersion::latest()).await;
+        aws_sdk_s3::Client::new(&config)
+    });
+
+    let mut group = c.benchmark_group("put_get_delete");
+    group.sample_size(10).confidence_level(0.99);
+
+    const KB: usize = 1024;
+    for size in [64 * KB, KB * KB].iter() {
+        let object: Vec<u8> = vec![0; *size];
+        group.bench_with_input(BenchmarkId::new("size", size), size, |b, _| {
+            b.to_async(Runtime::new().unwrap()).iter(|| async {
+                for i in 0..number_of_iterations {
+                    let bucket = &buckets[i % buckets.len()];
```

Comment:
**correctness**: This is round robin selecting a bucket each iteration which doesn't seem like what we want from trying to keep as many outside variables changing as possible when calculating the statistics.

Reply:
This round robin selection also comes from the internal benchmark doc.